### PR TITLE
Feature/ical support

### PR DESCRIPTION
**Pull request by P1tt187**

### CODE REVIEW

File: app/ClubEvent.php [hunk 1]
```diff
@@ -45,7 +45,8 @@ class ClubEvent extends Model
 								'evnt_time_end',
 								'evnt_public_info',		
 								'evnt_private_details',
-								'evnt_is_private');
+								'evnt_is_private',
+                                'evnt_is_published');
```
Comment (ThomDietrich) [inline app/ClubEvent.php]: Tabs vs Spaces :monkey:

Comment (P1tt187) [inline app/ClubEvent.php]: weared, in the ide it looks normal

Comment (ThomDietrich) [inline app/ClubEvent.php]: Some editors see tabs as two spaces, others as four, GitHub goes all nuts with 8. I've been haunted by the Spaces vs. Tabs dilemma. Use one or the other, just don't [mix](https://www.emacswiki.org/pics/static/TabsSpacesBoth.png) them :smiley: You might want to activate invisible characters in your IDE and make sure to use the formatter functionality to enforce some style guide.

Comment (P1tt187) [inline app/ClubEvent.php]: thx 👍

**Comment by P1tt187** [comment]
#209 integrate iCal Support

**Pull request merged**